build.gradle.kts

```diff
@@ -92,6 +99,15 @@ tasks.withType<KotlinCompile>().configureEach {
     }
 }
 
+tasks.withType<KotlinJsIrLink>().configureEach {
+    compilerOptions {
+        // Catching IndexOutOfBoundsException in Kotlin/Wasm is impossible by default,
```

Comment:
Could you please add some explanation here about why we need to be able to catch `IndexOutOfBoundsException`?

Reply:
Sure, added in https://github.com/charleskorn/kaml/pull/567/commits/f18b420eead46368f0cdb6b53a31924a9251ae40.